snowflake_blueprints/upload_file.py

```diff
@@ -51,23 +54,107 @@ def get_args():
         choices={
             'fail',
             'replace',
-            'append'},
+            'append'
+        },
         default='append',
         required=False)
+    parser.add_argument(
+        "--snowflake-data-types", dest="snowflake_data_types", required=False
+    )
     args = parser.parse_args()
 
     return args
 
 
-def create_table(source_full_path, table_name, insert_method, db_connection):
+def map_snowflake_to_pandas(snowflake_data_types):
+    """ Helper function to map a snowflake data type to the associated pandas data type
+
+    Args:
+        List 
+
+    Returns:
+        dict | None: Dict where the key is the field name and the value is the pandas data type
+    """
+    if snowflake_data_types is None:
```

Comment:
Same thing here re: None not actually existing if the user enters nothing in the UI.

Reply:
These have been addressed. I set the value to `None` if the args were left blank.
```
    if args.snowflake_data_types != '':
        data_types = ast.literal_eval(args.snowflake_data_types)
    else:
        data_types = None
```